corehq/apps/fixtures/tests/test_fixture_data.py

```diff
@@ -288,6 +290,24 @@ def test_indexed_global_fixture(self):
             ]
         )
 
+    def test_simulate_s3_blob(self):
+        config = settings.S3_BLOB_DB_SETTINGS
+        db = TemporaryS3BlobDB(config)
+        self.addCleanup(db.close)
+
+        sandwich = self.make_data_type("sandwich", is_global=True)
+        sandwich.save()
+        self.make_data_item(sandwich, "7.39")
+
+        fixtures = call_fixture_generator(self.user.to_ota_restore_user(self.domain))
```

Comment:
Does this actually fail without the changes in the previous commit?

Reply:
yes it does, it fails with the same error that we saw on production/staging

Reply:
awesome - wasn't sure exactly where that io read thing was happening, and whether it would be the same in the test version of this blobdb.